readme.md

```diff
@@ -1,70 +1,100 @@
+[Babel]: https://babeljs.io/
+[server]: https://github.com/webpack/webpack-dev-server
+[Jest]: https://jestjs.io/
 
 ### Requirements
 
-Webpack needs nodejs to do its thing. We aren't making a node app - just using it as a dev tool.
+To get started you'll need to make sure you have the following software tools installed:
 
-* [nodejs](https://nodejs.org/en/)
+- [Git](https://git-scm.com/)
+- [Node.js](https://nodejs.org/en/) (version 12 or higher is required)
+- [npm](http://npmjs.com/) (this should be installed along with Node.js)
 
-Installing node.js will also install [npm](https://www.npmjs.com) for us too. We'll need it for dependencies and some scripts.
+### Installation and setup
 
-### Dependencies
+1. Clone this Git repository and change to the new directory that has been created:
 
-We have specified all the dependencies we need in the `package.json` file. Npm will install them for us:
+    ```sh
+    git clone https://github.com/whichdigital/fe-techtest.git --depth 1
+    cd fe-techtest
+    ```
 
-`$ npm install`
+2. Install all of the project dependencies using npm:
 
-### Run!
+    ```sh
+    npm install
+    ```
 
-Ok, time to run the app. Npm takes care of business again:
+3. Once installed you can run the Webpack development server:
 
-`$ npm start`
+    ```sh
+    npm start
+    ```
 
-This will fire up the wepback dev server. It's not important to understand how it works for now.
+    Now open [http://localhost:8080/](http://localhost:8080/) in your browser and you should see a basic HTML page and if you open the developer console in the browser you should see a welcome message which confirms everything is loading and running correctly.
 
-Open [http://localhost:8080/](http://localhost:8080/) in your browser and you should see a basic html page. If you open the development console in your browser you should see a welcome message.
+    You will need to press `ctrl + c` to stop the server.
 
-### Development
+4. And to run your tests use the following command:
 
-The source files are located in `src/`. If you change a file, webpack will reload your files and refresh the browser for you.
+    ```sh
+    npm test
+    ```
 
-### Test
+## The exercise
 
-You can run the example tests:
+### Goal
 
-`$ npm test`
+We would like you to create a client-side web application to help a child understand multiplication. Your application will display a grid of numbers, when a child selects a number the multiples of that chosen number will be highlighted.
 
-## The exercise
+### Functional requirements
 
-__Goal__: Production of a client-side Web application to help a child understand multiplication.
+- The application will display numbers from 1 to 144.
+- When a number is clicked all numbers which are a multiple of the chosen number must be highlighted.
+- When a new number is chosen the previously highlighted numbers must be updated.
 
-__Duration__: This test is intended to take ~2 hours but may take more
```

Comment:
I'm not sure I liked the way it was written before, but I do like having an idea of having an expectation of time spent. I'm guessing you discussed this and decided to remove it? 

Could phrase it something like `You should spend a maximum of about two hours on this task`?

Reply:
The worry was that by stating an expected time somebody is more likely to spend that amount of time on it - which is anecdotally true.

Instead, we plan to update the guidance that our recruitment team give candidates depending on their experience.

Reply:
That makes a lot of sense!

Reply:
That makes a lot of sense!